airflow/config/webserver_entry.sh

```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env bash
 
 set -Eeuxo pipefail
-
-airflow initdb
+airflow db init
```

Comment:
I don't get why this `webserver_entry.sh` is so different from the one we already have? What happened?

The `webserver_entry.sh` was already modernised for Airflow 2 and fixed several issues. So this should revert to the version we are already using.